assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/ActivateAnalyticsCTA.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * ActivateAnalyticsCTA component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useActivateModuleCallback from '../../../../../hooks/useActivateModuleCallback';
+import Button from '../../../../../components/Button';
+import PreviewGraph from './PreviewGraph';
+import VisitorsGraph from '../../../../../../svg/graphics/cta-graph-visitors.svg';
+import GoalsGraph from '../../../../../../svg/graphics/cta-graph-goals.svg';
+
+export default function ActivateAnalyticsCTA() {
+	const activateModuleCallback = useActivateModuleCallback( 'analytics' );
+
+	if ( ! activateModuleCallback ) return null;
+
+	return (
+		<div className="googlesitekit-cta--analytics googlesitekit-cta--analytics--full">
```

Comment:
Hi @asvinb, thanks for your comment. At first glance, this is a logical suggestion, and I actually started off writing this using `Grid/Row/Cell` components. However, the grid system doesn't play so nicely with the design in this case. Notably, the requirement to lay out the cells in reverse depending on the layout: 

_Narrow viewport, the graphs are first in the flow:_
![image](https://user-images.githubusercontent.com/18395600/155140639-48e52e29-94e5-49a3-8f74-26d4a0d7b2b7.png)

_Wider viewport, the description/button are first in the flow:_
![image](https://user-images.githubusercontent.com/18395600/155140721-2054f2f5-7e22-4520-8b67-bf211dc4046c.png)

Getting this to work with our grid system is problematic, as CSS grid doesn't have an equivalent of Flexbox's `wrap-reverse`, and the way the grid spans are defined doesn't lend itself to easily reversing the order of spanned cells. I think, even if we were to achieve this with the grid layout, we'd need to override it to an extent that it would be quite brittle and overly complex compared to the Flexbox solution.

By all means, take a look and see if you can figure out an elegant solution using our grid, but that's been my experience to date.

Reply:
> I would also rename the class name from googlesitekit-cta--analytics to googlesitekit-analytics-cta to be consistent with the component. Then the class name for the full variant would be: googlesitekit-analytics-cta--full

That sounds good to me, and I will make that change :+1:

Reply:
@techanvil So we can use the `Grid/Row/Cell` components to achieve the layout and set the order per breakpoint. If you look at the `Cell` [component](https://github.com/google/site-kit-wp/blob/develop/assets/js/material-components/layout/Cell.js#L38), you can set the order of the cell using the following props: 

- `smOrder`
- `mdOrder`
- `lgOrder`

I tested it quickly and it worked great. However the only reason right now where it'll be cumbersome to use if is that we only have styles for the `sm`, `md` and `lg` breakpoints where `lg` is for `> 961px`. Thus we cannot target higher breakpoints using the `Grid` components, for e.g we don't have something like `xlSize` where we could potentially set the size at let's say `1280px` or `1440px`. So we are kinda limited in this case especially how the component is designed.
That being said, happy to keep the flexbox solution in this case, but from a UI point of view and looking at the Analytics data based on screen resolutions, around 30% of the users will see the following layout where we have lots of whitespace:
<img width="1152" alt="image" src="https://user-images.githubusercontent.com/31818/155326198-5aeb82c3-28ed-4761-b030-333a8fc4669c.png">
Screenshot was taken at 1366x768.
I'am thinking maybe till `1536px` we can have the following layout instead:
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/31818/155326948-7344a705-f104-4aaa-8be8-4b04f3392eec.png">
where everything is stacked. In the Figma designs, the plugin area only is set to `1440px` which does not take into account the WordPress left menu.
@marrrmarrr Let me know what you think.

Reply:
Thanks @asvinb and @marrrmarrr.

> @techanvil So we can use the Grid/Row/Cell [component](https://github.com/google/site-kit-wp/blob/develop/assets/js/material-components/layout/Cell.js#L38)s to achieve the layout and set the order per breakpoint. If you look at the Cell component, you can set the order of the cell using the following props:
> * smOrder
> * mdOrder
> * lgOrder

@asvinb, thanks for pointing this out! I have seen these props in the past, but not actually worked directly with them, so this is a handy reminder, I will keep it in mind for future grid work.

As you've noticed though, there are other factors to consider e.g. the larger viewport and so I still remain happy to keep the Flexbox approach, if you are also so inclined.

I have adjusted the responsive layout to avoid that double-stacked preview graph, so the desktop layout now looks like this from 961px (desktop):
![image](https://user-images.githubusercontent.com/18395600/155386673-f5ff3542-635a-4dab-b2db-5e268968fbf5.png)

Then, from 1280px (xlarge):
![image](https://user-images.githubusercontent.com/18395600/155386861-ae03af45-b368-4eb6-9235-2d4be3b4d1c5.png)

And from 1536px:
![image](https://user-images.githubusercontent.com/18395600/155387409-5ce9e1fc-3810-410e-a133-b6954411a09b.png)

Reply:
Looks good @techanvil !